google_guest_agent/oslogin.go

```diff
@@ -103,12 +103,22 @@ func (o *osloginMgr) set() error {
 		logger.Errorf("Error updating group.conf: %v.", err)
 	}
 
-	for _, svc := range []string{"ssh", "sshd", "nscd", "unscd", "systemd-logind", "cron", "crond"} {
+	for _, svc := range []string{"nscd", "unscd", "systemd-logind", "cron", "crond"} {
```

Comment:
We have to restart sshd if oslogin is disabled (sshd configs change). So we still need to keep this here as well right?

Reply:
ssh is still being restarted, it was only moved down below this